scipy/stats/_multivariate.py

```diff
@@ -847,16 +904,20 @@ def rvs(self, alpha, size=1):
 
         """
         alpha = _dirichlet_check_parameters(alpha)
-        return np.random.dirichlet(alpha, size=size)
+        if random_state is None:
+            return self.random_state.dirichlet(alpha, size=size)
```

Comment:
Yes.
Correct me if I'm wrong: using the property here incurs an overhead of `__get__`, which can be avoided. 
Granted, this is a minor quibble.